docs/user/setup.rst

```diff
@@ -86,14 +80,14 @@ installation of Arch Linux.
 .. code-block:: shell
 
     $ sudo dnf install llvm clang
-    $ sudo dnf install libunwind-devel gc-devel zlib-devel re2-devel # optional
+    $ sudo dnf install libunwind-devel gc-devel zlib-devel # optional
```

Comment:
We don't need libunwind deps any more either.

Reply:
Done.  Built on my local machine & viewed in Firefox.

I also searched  all the .rst file in the docs tree for the case-insensitive string "unw".
That showed a few references, but nothing that I could see related to the full 
libunwind.

Reply:
As long as I was in setup.html making non re2s related changes, it seemed fair game
for me to change the description of the default garbage collector to 'immix'. I also
provided a hyperlink to the original paper and did some rewording.

All builds & views. Hyperlink from setup.rst to intended section of sbt.rst works.